kss-config.json

```diff
@@ -26,6 +26,7 @@
     "public/js/components/file.js",
     "public/js/components/breadcrumb.js",
     "public/js/components/collapse.js",
+    "public/js/components/modal.js",
```

Comment:
Generally speaking, keep including bootstrap components is not a good practice. We should move out of boostrap, basically, and components are the first thing supposed to "disappear" as a dependency. Couldn't we do this simply with jquery ui (which is not supposed to be removed anytime soon)